src/systemcmds/tests/test_dataman.c

```diff
@@ -63,6 +63,16 @@ int test_dataman(int argc, char *argv[]);
 
 #define NUM_MISSIONS_TEST 50
 
+typedef union dataman_max_size_t {
+	struct home_position_s		safe_item;
+	struct mission_item_s		mission_item;
+	struct mission_s			mission;
+	struct fence_vertex_s		vertex;
+	struct dataman_compat_s		compat;
+} dataman_max_size_t;
+
+#define DM_MAX_DATA_SIZE sizeof(dataman_max_size_t)
```

Comment:
Well this buffers is used to read more than one type of dm_item_t but if you think is better I just pushed a version changing to struct mission_s and also another patch that check for the result of each task.